roadmap/implementers-guide/src/runtime/router.md

```diff
@@ -260,6 +260,7 @@ any of dispatchables return an error.
 Utility routines.
 
 `queue_downward_message(P: ParaId, M: DownwardMessage)`:
+    1. Check if the serialized size of `M` exceeds the `config.critical_downward_message_size`. If so, return an error.
```

Comment:
What would be the consequence of this error? Silently dropping the message?

Reply:
This utility routine is supposed to be called only by the runtime, so basically the caller will be notified about the error. I'd expect that the caller should make sure that it doesn't try to send anything that will not fit into the queue.